### PR TITLE
fixes #98, variable dictionaries in mixins are valid

### LINKED ISSUE #98: False error on placeholder function
This one is a _major blocker_ for us: gozales-pe throws an error when it encounters LESS placeholder functions. These are perfectly valid and pass linting with `lessc --lint`.

Test Script:

```
var gonzales = require('gonzales-pe');
var less = '.transition(...){}';
var result = gonzales.parse(less, {
    syntax: 'less'
});
console.log(result);
```

Output:

```
→ node lesstest.js                  

/.../node_modules/gonzales-pe/lib/parse.js:5
= parse(tokens,rule,needInfo);}catch(e) {if(!e.syntax)throw e;throw new Parsin
                                                                    ^
Parsing error: Please check validity of the block starting from line #1
```

When `var less = '.transition(...){}';` is changed to `var less = '.transition(){}';` gonzales-pe allows it to pass. That should not be required, as the former value is completely valid.

### PR DESCRIPTION
**Pull request by shellscape**

### CODE REVIEW

**Comment by tonyganch** [comment]
First of all, thank you very much for all those pull requests!
If I don't have enough time to look them through today, I'll definitely find a minute on weekend.
Please don't be mad at me if I point at some errors, I still think that you've made an amazing job.

File: src/less/parse.js [hunk 1]
```diff
@@ -182,11 +182,15 @@ function getLastPositionForArray(content, line, column, colOffset) {
...
+    if (tokens[pos - 1].type !== 'Newline') {
+      position[1] += colOffset;
+    } else {
+      position[0]++;
+      position[1] = 1;
+    }
   } else {
-    position[0]++;
-    position[1] = 1;
+    position[1] = colOffset;
```
Comment (tonyganch) [inline src/less/parse.js:193]: Can you explain please why you've made this change?

Comment (shellscape) [inline src/less/parse.js:193]: tokens[pos - 1] is undefined while tests run. under that circumstance, the column position is equal to the colOffset for a variable dictionary. tested that with real-world examples (as the test less files aren't really) and it checked out.

**Comment by shellscape** [comment]
This fixes a pretty **critical oversight and breaking bug** in the 3.x versions that worked fine with 3.0.0.-31. At the very least, with all tests passing, it should be merged and "corrected" in a patch version.

**Comment by shellscape** [comment]
abandoning, moved on to use PostCSS and https://github.com/webschik/postcss-less.

**Pull request closed without merging**